### ISSUE TITLE
kodi.GetNextUnwatchedEpisode() doesn't work right

### ISSUE THREAD

**Issue by jingai** [m0ngr31/kodi-alexa#151]
It seems to return something even if there are no unwatched episodes.  In this case, it should return nothing at all so the caller can decide to do something else.

Further, when there are unwatched episodes, it doesn't seem to always return the next unwatched according to Kodi's own watched status indicators.

I haven't really spent much time debugging the odd behavior, so I can't be more specific right now.  I'll look into it further later though if no one else has a look at this.

**Comment by jingai** [comment]
I don't see this or any other odd behavior for movies.

**Comment by mcl22** [comment]
For tv shows I see this problem very often. When I watch an episode and stop playback after a couple of minutes the show keeps unwatched in the kodi interface (as intended). But if I now use WatchNextEpisode it will play the next one instead of the one I only watched for a few minutes. 
And if you stop this episode now and call WatchNextEpisode it will again play the next one and so on.

If there are no unwatched episodes I get different results. For some shows Alexa tells me that there a no unwatched episodes, for others she starts some episode that I already watched.

One last thing: I recognized that the WatchLatestEpisode seems to refer to the creation date of a file. Perhaps this is intended? But the air date would be an alternative :)
